### PR TITLE
Add alias and remove duplicate files

### PR DESCRIPTION
**Pull request by weh**
## Double check these details before you open a PR

- [x] PR does not match another non-stale PR currently opened

## Features

This solves a part of the open tasks from https://github.com/devicons/devicon/pull/1998#issuecomment-1807542194

* [x]  digitalocean (#2005)
* [x]  gatsby (#2006)
* [x]  mysql (#2007)
* [x]  nodejs (#2011)
* [x]  opensuse (#2002)
* [x]  postcss (#2012)
* [ ]  spring
* [ ]  typo3

## Notes

I will check on the opensuse icons later. the svg files currently include inkscape metadata. either there is a open PR or i add it later

### CODE REVIEW

**Comment by weh** [comment]
Oh, btw this is the PR where i asked myself if we want to have newlines in the svg files, or if we need to remove them.

File: icons/gatsby/gatsby-plain-wordmark.svg
Comment (canaleal) [inline icons/gatsby/gatsby-plain-wordmark.svg:1]: The `gatsby-original-wordmark.svg` has 2 paths and 2 different fill colors. It unfortunately cant be used for `plain-wordmark`.

File: icons/digitalocean/digitalocean-plain-wordmark.svg
Comment (canaleal) [inline icons/digitalocean/digitalocean-plain-wordmark.svg:1]: Thankfully both plain svgs for digital ocean can be removed, but the original svgs have to be updated to have 1 path and fill color. They also have to be scaled to fit the viewbox.

image

**Review by canaleal — CHANGES_REQUESTED** [review]
Some minor modifications need to be made for `digitalocean`, `postcss`, and `gatsby`, but aside from that all is good.

**Comment by lunatic-fox** [comment]
> Oh, btw this is the PR where i asked myself if we want to have newlines in the svg files, or if we need to remove them.

@weh, the newline character at the end of the file is used, if I'm not mistaken, for compatibility reasons.

**Comment by lunatic-fox** [comment]
First of all, thank you @weh and @canaleal for quickly make pull requests to fix the found errors. That means a lot! 🚀

@weh, as @canaleal had explained in this [comment](https://github.com/devicons/devicon/pull/2001#discussion_r1391798515), we need to make some changes in the icons.

This type of error needs a pull request for each icon, like the one you made for [opensuse](https://github.com/devicons/devicon/pull/2002), because even if we are not modifying an icon, some icon will be deleted eventually. I'm not sure, but since pull request titles together with `devicon.json` affect what icons will be built I think this is a safer strategy.

**Comment by weh** [comment]
sure, i will split the request, and also fix the mentioned svg files when i find time :+1:

File: icons/digitalocean/digitalocean-plain-wordmark.svg
Comment (weh) [reply icons/digitalocean/digitalocean-plain-wordmark.svg:1]: done in https://github.com/devicons/devicon/pull/2005

File: icons/gatsby/gatsby-plain-wordmark.svg
Comment (weh) [reply icons/gatsby/gatsby-plain-wordmark.svg:1]: updated in #2001

File: icons/postcss/postcss-plain.svg
[the remark this answers is not in the document]
Comment (weh) [reply icons/postcss/postcss-plain.svg:1]: updated in #2012

**Comment by Snailedlt** [comment]
Closing this PR since it's being split into multiple PR's

**Pull request closed without merging**